docs/notebooks/word_to_sentence_embedding.ipynb

```diff
@@ -0,0 +1,1264 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gensim.models import Word2Vec, KeyedVectors, TfidfModel\n",
+    "from gensim.parsing.preprocessing import STOPWORDS\n",
+    "from scipy.sparse.linalg import svds\n",
+    "from scipy.spatial.distance import cosine\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using pre-tained word2vec model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Download the pretrained model form [here](https://github.com/RaRe-Technologies/gensim-data/releases/tag/glove-wiki-gigaword-200)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
```

Comment:
Cells was run chaotically (something not running at all, something running in strange order)

Reply:
Less RAM, can't load all these model on a single run. I'll try this at the end.